app/config/config.test.json

```diff
@@ -32,6 +32,7 @@
 	"membership-application-timeframe-limit": "PT10M",
 	"operator-email": "email@operatorsownmailserver.com",
 	"operator-displayname": "Friendly Operator",
+	"operator-displayname-alternative": "Alternative Operator",
```

Comment:
Well now I know why I got 50 emails or so from a "	Alternative Operator" in my inbox...